src/main/java/io/seqera/tower/cli/responses/RunView.java

```diff
@@ -48,92 +133,109 @@ public void toString(PrintWriter out) {
-                ++manifestCount;
-            }
-
-            if (manifestCount > 0) {
-                out.println(ansi(String.format("%n%n    @|bold Manifest|@")));
-                manifestTable.print();
-            }
-        }
-
-        if (workflow.getStats() != null) {
-            out.println(ansi(String.format("%n%n    @|bold Status|@")));
-
-            TableList statsTable = new TableList(out, 2);
-            statsTable.setPrefix("    ");
-            statsTable.addRow("Pending", workflowLoad.getPending().toString());
-            statsTable.addRow("Submitted", workflowLoad.getSubmitted().toString());
-            statsTable.addRow("Running", workflowLoad.getRunning().toString());
-            statsTable.addRow("Cached", workflowLoad.getCached().toString());
-            statsTable.addRow("Succeeded", workflowLoad.getSucceeded().toString());
-            statsTable.addRow("Failed", workflowLoad.getFailed().toString());
-            statsTable.print();
-        }
-
-
-        if (workflow.getParams() != null && !workflow.getParams().isEmpty()) {
+        if (!config.isEmpty()) {
+            out.println(ansi(String.format("%n    @|bold Configuration|@")));
+            out.println(ansi(config.toString()));
+        }
+
+        if (!params.isEmpty()) {
             out.println(ansi(String.format("%n    @|bold Parameters|@")));
-            TableList paramsTable = new TableList(out, 2);
-            paramsTable.setPrefix("    ");
-            workflow.getParams().forEach((key, value) -> paramsTable.addRow(key, value.toString()));
+            out.println(ansi(params.toString()));
         }
 
-        if (workflow.getConfigText() != null && !workflow.getConfigText().isBlank()) {
-            out.println(ansi(String.format("%n    @|bold Configuration|@")));
-            out.println(workflow.getConfigText());
+        if (command != null) {
+            out.println(ansi(String.format("%n    @|bold Command|@")));
+            out.println(ansi(String.format("%n    %s", command)));
+        }
+
+        if (!status.isEmpty()) {
+            out.println(ansi(String.format("%n    @|bold Status|@")));
+            TableList tableStatus = new TableList(out, 2);
+            tableStatus.setPrefix("    ");
+            tableStatus.addRow("Pending", status.get("pending").toString());
+            tableStatus.addRow("Submitted", status.get("submitted").toString());
+            tableStatus.addRow("Running", status.get("running").toString());
+            tableStatus.addRow("Cached", status.get("cached").toString());
+            tableStatus.addRow("Succeeded", status.get("succeeded").toString());
+            tableStatus.addRow("Failed", status.get("failed").toString());
+            tableStatus.print();
+        }
+
+        if (!processes.isEmpty()) {
+            out.println(ansi(String.format("%n    @|bold Processes|@")));
+            TableList tableProcess = new TableList(out, 2, "Name", "Completed");
```

Comment:
Some processes may not be completed, which is why there is a completed column

Reply:
This is why I suggest to change the title to `Completed processes` and then you see `3/5` (reads three completed processes of five). It seems clear to me. Do you think it may be confusing?